### PR TITLE
Make sure stdin can also be used

### PR DESCRIPTION
**Pull request by pieterdeschepper**

### CODE REVIEW

File: budou/budou.py [hunk 1]
```diff
@@ -58,9 +58,16 @@ def main():
   if args['--version']:
     print(__version__)
     sys.exit()
+    
+  if args['<source>']:
+    source = args['<source>']
+  elif not sys.stdin.isatty():
+   source = sys.stdin.read()
```
Comment (tushuhei) [inline budou/budou.py]: Please add one space to align the indent.

File: budou/budou.py [hunk 2]
```diff
@@ -58,9 +58,16 @@ def main():
...
+  else:
+    sys.exit()
```
Comment (tushuhei) [inline budou/budou.py]: Could you add another logic to show the command help if the source is empty?

File: budou/budou.py (hunk 1 above)
Comment (pieterdeschepper) [reply budou/budou.py]: Fixed

File: budou/budou.py (hunk 2 above)
Comment (pieterdeschepper) [reply budou/budou.py]: Done

**Pull request merged**